<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialogClass</class>
 <widget class="QDialog" name="SettingsDialogClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>373</width>
        <height>344</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Schemes</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QMaemo5ValueButton" name="schemeButton">
         <property name="text">
          <string>Current Scheme</string>
         </property>
         <property name="valueLayout" stdset="0">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QMaemo5ValueButton" name="midiConnButton">
         <property name="text">
          <string>MIDI Connection</string>
         </property>
         <property name="valueText" stdset="0">
          <string/>
         </property>
         <property name="valueLayout" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QMaemo5ValueButton" name="netcastButton">
         <property name="text">
          <string>Audio In Connection</string>
         </property>
         <property name="valueText" stdset="0">
          <string/>
         </property>
         <property name="valueLayout" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Options</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QMaemo5ValueButton" name="advSettingsButton">
         <property name="text">
          <string>Open Advanced Settings</string>
         </property>
         <property name="valueText" stdset="0">
          <string>For tunning MIDI and Audio settings</string>
         </property>
         <property name="valueLayout" stdset="0">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QMaemo5ValueButton" name="homepageButton">
         <property name="text">
          <string>Open Homepage</string>
         </property>
         <property name="valueText" stdset="0">
          <string>http://midij.home.sapo.pt</string>
         </property>
         <property name="valueLayout" stdset="0">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="buttonBoxLayout">
     <item>
      <spacer name="buttonBoxSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMaemo5ValueButton</class>
   <extends>QPushButton</extends>
   <header>../gui/GuiUtils.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>homepageButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>launchHomepage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advSettingsButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>settingsSwap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>launchHomepage()</slot>
  <slot>settingsSwap()</slot>
 </slots>
</ui>
